ReactWindows/ReactNative.Net46/packages.config

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Facebook.Yoga" version="1.0.1-pre" targetFramework="net46" />
+  <package id="Facebook.Yoga" version="1.0.2-pre" targetFramework="net46" />
   <package id="Microsoft.ChakraCore" version="1.4.1-preview-00010-42060" targetFramework="net46" developmentDependency="true" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net46" />
   <package id="PCLStorage" version="1.0.2" targetFramework="net46" />
-  <package id="Postal.NET" version="1.0.0.1" targetFramework="net46" />
+  <package id="Postal.NET" version="1.0.0.6" targetFramework="net46" />
   <package id="SocketMessaging" version="1.1.6002.26530" targetFramework="net46" />
```

Comment:
It looks like this has been here for a while, but never included in the .csproj.  Can we just remove this dependency?